src/api.ts

```diff
@@ -373,35 +385,33 @@ export function touchCommit( hash: CommitHash ) {
 	state.accesses.set( hash, Date.now() );
 }
 
+export function touchContainer( name: ContainerName ) {
+	state.accessesByContainer.set( name, Date.now() );
+}
+
 export function getCommitAccessTime( hash: CommitHash ): number | undefined {
 	if ( ! hash ) {
 		return undefined;
 	}
 	return state.accesses.get( hash );
 }
 
+export function getContainerAccessTime( name: ContainerName ): number | undefined {
+	if ( ! name ) {
+		return undefined;
+	}
+	return state.accessesByContainer.get( name );
+}
+
 /*
- * Get all currently running containers that were created by dserve and have expired.
+ * Get all currently running containers that have expired.
  * Expired means have not been accessed in EXPIRED_DURATION
  */
-export function getExpiredContainers(
-	containers: Array< ContainerInfo >,
-	getAccessTime: Function
-) {
-	// if the server is newly spun up, wait a bit before killing off running containers
-	if ( Date.now() - START_TIME < TEN_MINUTES ) {
```

Comment:
This logic was duplicated in https://github.com/Automattic/dserve/blob/master/src/index.ts#L277.

I removed it from here and updated the timer to `TEN_MINUTES` in `index.ts`